test/integration/test_kraken_order_book_tracker.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+import math
+import time
+from os.path import join, realpath
+import sys; sys.path.insert(0, realpath(join(__file__, "../../../")))
+
+from hummingbot.core.event.event_logger import EventLogger
+from hummingbot.core.event.events import (
+    OrderBookEvent,
+    OrderBookTradeEvent,
+    TradeType
+)
+import asyncio
+import logging
+from typing import (
+    Dict,
+    Optional,
+    List)
+import unittest
+
+from hummingbot.core.data_type.order_book import OrderBook
+from hummingbot.core.data_type.order_book_tracker import OrderBookTrackerDataSourceType
+from hummingbot.core.utils.async_utils import (
+    safe_ensure_future,
+    safe_gather,
+)
+from hummingbot.market.kraken.kraken_order_book_tracker import KrakenOrderBookTracker
+
+
+class KrakenOrderBookTrackerUnitTest(unittest.TestCase):
+    order_book_tracker: Optional[KrakenOrderBookTracker] = None
+    events: List[OrderBookEvent] = [
+        OrderBookEvent.TradeEvent
+    ]
+    trading_pairs: List[str] = [
+        "XBT/USD",
+        "XRP/USD"
+    ]
```

Comment:
Looking at the output of `get_trading_pairs()` and the `trading_pair` here, there is a definite difference. It would be best to stick with one convention